### ISSUE TITLE
Provide for docker build and install script

### ISSUE THREAD

**Issue by girish17** [girish17/op-mattermost#5]
so that one line install is supported through a docker command execution

**Comment by alexanderadam** [comment]
Is [this](https://hub.docker.com/r/girish17/op-mattermost/tags) the official image?
Are there any plans to have automatic builds by version tags (i.e. like [this](https://docs.docker.com/docker-hub/builds/) or via [GitHub Actions](https://docs.docker.com/ci-cd/github-actions/))?

**Comment by girish17** [comment]
Yes, that's the docker image. The docker images are automatically built on every code commit. GitHub Actions can be tracked here - https://github.com/girish17/op-mattermost/actions

Added @alexanderadam as collaborator.

**Comment by girish17** [comment]
A year ago the docker auto deploy worked fine. Now, the automatic deployment on docker hub needs Pro subscription. Currently, manual docker build is supported.

**Comment by mmospanenko** [comment]
will be great to have versioned images on Docker Hub
